packages/devtools_app/lib/src/shared/console/widgets/help_dialog.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../../shared/analytics/constants.dart' as gac;
+import '../../common_widgets.dart';
+
+class ConsoleHelpLink extends StatelessWidget {
+  const ConsoleHelpLink({Key? key}) : super(key: key);
+
+  static const _documentationTopic = gac.consoleHelp;
+
+  HelpButtonWithDialog _createButton() => HelpButtonWithDialog(
+        asAction: true,
+        gaScreen: gac.memory,
+        gaSelection: gac.topicDocumentationButton(_documentationTopic),
+        dialogTitle: 'Console Help',
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.end,
+          children: [
+            const Text(
+              r'''
+Use debug console to:
+
+1. Watch the standard output (stdout) of the application.
+2. Evaluate expressions for a paused or running application.
+3. Analyze inbound and outbound references for objects, dropped from memory heap snapshots.
+
+Assign previously evaluated objects to variable
+using $0, $1 … $5.
+Example: var x = $0
```

Comment:
use RichText instead of text so that `$0` values and `var x = $0` can be styled in a fixed font style